### PR TITLE
Move common distro parts to include file to reduce code duplication

### PR DESCRIPTION
**Pull request by m-iwanicki**
About `DISTRO_VERSION:append = ""` we can either add sub-version to main one or remove it altogether and keep one version for both distros otherwise it might be missed.

### CODE REVIEW

File: meta-zarhus-distro/conf/distro/include/zarhus-distro-common.conf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require conf/distro/poky.conf
+
+# distro name
+DISTRO_NAME = "Distro for Zarhus product"
```
Comment (DaniilKl) [inline meta-zarhus-distro/conf/distro/include/zarhus-distro-common.conf]: I think `DISTRO_NAME` as well as `DISTRO` should be explicitly defined for every distro configuration, it is not a common config, so default `DISTRO_NAME` is rather surplus here.

File: meta-zarhus-distro/conf/distro/include/zarhus-distro-common.conf [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+DISTRO_VERSION = "0.1.0"
+SDK_VENDOR = "-zarhussdk"
+
+MAINTAINER = "3mdeb Sp. z o. o. <contact@3mdeb.com>"
+
+TARGET_VENDOR = "-zarhus"
+
+# distro features
+# DISTRO_FEATURES_DEFAULT ?= "acl alsa argp bluetooth ext2 ipv4 ipv6 largefile pcmcia usbgadget usbhost wifi xattr nfs zeroconf pci 3g nfc x11 vfat"
```
Comment (DaniilKl) [inline meta-zarhus-distro/conf/distro/include/zarhus-distro-common.conf:10]: Why this one is commented out?

Comment (DaniilKl) [reply meta-zarhus-distro/conf/distro/include/zarhus-distro-common.conf:10]: I see it is defined in `poky/meta/conf/distro/include/default-distrovars.inc`, nvm then.

File: meta-zarhus-distro/conf/distro/zarhus-distro-webkit.conf [hunk 3]
```diff
@@ -1,37 +1,11 @@
...
-# mkpasswd should be used in order to set account password, this password for
-# root was created with the meta-zarhus layer using the following command:
-# `printf "%q" $(mkpasswd -m sha256crypt $ROOT_PASSWORD)`
-# TODO: add way to automate generating ROOT_PASSWD, maybe split this before vs
-# after honister update, because the method changed
-ROOT_PASSWD = "TODO"
 
+# We can add suffix to main version if only Webkit distro changes e.g.
+# 0.1.0-1.0.0.
+DISTRO_VERSION:append = ""
```
Comment (DaniilKl) [inline meta-zarhus-distro/conf/distro/zarhus-distro-webkit.conf]: This one should follow release process https://github.com/zarhus/docs/pull/5, and I do not see anything related there. Actually, IIUC, the release process assumes we have one distro for a repo, and do not cover multiple distros. Should we make different releases for different distros in one repo?

Comment (m-iwanicki) [reply meta-zarhus-distro/conf/distro/zarhus-distro-webkit.conf]: Probably best option would be to just have one version for all distros even if only Webkit part changed.

File: meta-zarhus-distro/conf/distro/include/zarhus-distro-common.conf (hunk 2 above)
Comment (m-iwanicki) [reply meta-zarhus-distro/conf/distro/include/zarhus-distro-common.conf:10]: zarhus-distro-common.conf is almost 1-to-1 copy of zarhus-distro.conf

File: meta-zarhus-distro/conf/distro/zarhus-distro-webkit.conf (hunk 3 above)
Comment (DaniilKl) [reply meta-zarhus-distro/conf/distro/zarhus-distro-webkit.conf]: So, lets define `DISTRO_VERSION` inside `zarhus-distro-common.conf`.

Comment (m-iwanicki) [reply meta-zarhus-distro/conf/distro/zarhus-distro-webkit.conf]: https://github.com/zarhus/meta-zarhus/pull/34/commits/d751e27a64a8939654ed89f1abd1b79ecc869214

File: meta-zarhus-distro/conf/distro/include/zarhus-distro-common.conf (hunk 1 above)
Comment (m-iwanicki) [reply meta-zarhus-distro/conf/distro/include/zarhus-distro-common.conf]: d824d1220d7508a05b724543d70038b6f017e707

**Review by DaniilKl — APPROVED** [review]

**Pull request merged**